packages/builder/src/components/start/CreateAppModal.svelte

```diff
@@ -207,16 +190,12 @@
         }}
       />
     {/if}
-    <Body size="S">
-      Give your new app a name, and choose which groups have access (paid plans
-      only).
-    </Body>
     <Input
       bind:value={$values.name}
       error={$touched.name && $errors.name}
       on:blur={() => ($touched.name = true)}
       label="Name"
+      placeholder={`${$auth.user.firstName}'s first app`}
```

Comment:
`firstName` might not be defined - so we might need to account for that and render some fallback like `Your first app` if that's the case.

Reply:
good call